### DISCUSSION TITLE
2021年の振り返りレポートをWebで見れますか？

### DISCUSSION THREAD

**Discussion by uhooi** [increments/qiita-discussions#100] [ヘルプ] [answered]
私は「Qiita からのお知らせ」で「メール」を OFF にしているためか、2021 年の振り返りレポートがメールで届きませんでした。
Web 上で見ることは可能でしょうか？
ざっと探した限りでは見つかりませんでした。

**Comment by kskwtnk** [comment, marked as answer]
@uhooi 

ご意見ありがとうございます！

仰るとおり、お知らせのメールをオフにしている方には送っておらず、なおかつWebでの提供は今年は予定しておりませんでした 🙏 
しかしTwitter上でも「自分も見たい！」という声をいただいているため追加送信を考えています

- フォームからQiitaのidを送信
- メールのお知らせをオン

の2点に対応いただけた方には、年明けになってしまいますがお届けしたいと思っています！

フォームの準備の完了や、日程が決まり次第アナウンス用のDiscussionを立てます！

**Reply by kskwtnk** [reply]
@uhooi 

すみません！社内でのやり取りの都合上、フォーム展開含め来年になってしまいそうです 🙏 
お待ちいただけると幸いです 🙇

**Reply by uhooi** [reply]
@xrxoxcxox 
迅速なご回答ありがとうございます 🙇 
もろもろ承知しました！
メールのお知らせを ON にしました。
来年のフォーム展開、お待ちしております。

**Reply by kskwtnk** [reply]
@uhooi 

お待たせしております、フォームを作成しました！
こちらから送信お願いします！

https://forms.gle/eRSHXdxSRodFeaYj9

若干話が逸れるのですが……共有です 🙏 

当初はメール通知をオンにしていただいた上で申し込みをしていただくつもりでいましたが、お手を煩わせてしまうので通知設定がオフのままでも今回リクエストいただいた方にはメールを送ることにしました
uhooiさんももし「通知自体はオフのままにしておきたかった……」ということであれば、再度オフにしていただいてもレポートをお届けします 🙇

**Reply by uhooi** [reply]
@xrxoxcxox ありがとうございます、回答しました！
メールにしか来ないお知らせを見逃したくないので、通知は ON のままにしておきますね！

**Comment by munierujp** [comment]
今回の機能とは別でもいいので、ウェブ上でアナリティクス的なものが見れると面白いかもしれません（Googleアナリティクスの設定とは別に）

**Reply by kskwtnk** [reply]
@munierujp 

ありがとうございます！
社内でも話が出たことはある、くらいのレベルなのですぐに実装には取りかかれなさそうですが……実際の声として共有いたします！
